Broken build, zbar.h: No such file or directory

This is caused by commit https://github.com/mchehab/zbar/commit/d09afa67a6df7c2ecbaab877d0630201799cd425

First I had a problem where it told me `ZBAR_SQCODE` is undefined, which I traced back to it using my system `zbar.h`. After I uninstalled my system zbar it complained about not finding `zbar.h`.

```
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../../zbar -I../include -I../../zbar -I../include -mthreads -I/mingw32/include -Wall -Wno-parentheses -g -O2 -MT libzbar_la-symbol.lo -MD -MP -MF .deps/libzbar_la-symbol.Tpo -c ../../zbar/symbol.c  -DDLL_EXPORT -DPIC -o .libs/libzbar_la-symbol.o
../../zbar/config.c:31:10: fatal error: zbar.h: No such file or directory
 #include <zbar.h>
          ^~~~~~~~
compilation terminated.
```

--- a/zbar/Makefile.am
+++ b/zbar/Makefile.am
@@ -1,5 +1,5 @@
 lib_LTLIBRARIES = libzbar.la
-libzbar_la_CPPFLAGS = -I$(srcdir) $(AM_CPPFLAGS)
+libzbar_la_CPPFLAGS = -I$(srcdir) -I$(srcdir)/../include $(AM_CPPFLAGS)
 libzbar_la_LDFLAGS = -no-undefined -version-info $(LIB_VERSION) \
     -export-symbols-regex "^(zbar|_zbar.*_error)_.*" $(AM_LDFLAGS)
 libzbar_la_LIBADD = $(LTLIBICONV)
@@ -57,6 +57,15 @@ libzbar_la_SOURCES += sqcode.h sqcode.c \
 endif
 
 if WIN32
+
+%-rc.o: %.rc
+	$(RC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) \
+	$(AM_CPPFLAGS) $(CPPFLAGS) -o $@ $<
+%-rc.lo: %.rc
+	$(LIBTOOL) --tag=RC $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) \
+	--mode=compile $(RC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) \
+	$(AM_CPPFLAGS) $(CPPFLAGS) -o $@ $<
+
 libzbar_la_SOURCES += processor/win.c libzbar.rc
 libzbar_la_CPPFLAGS += -mthreads
 libzbar_la_LDFLAGS += -mthreads

